documentation/Maintenance.md

```diff
@@ -745,6 +745,8 @@ To change the operating system on an already running cluster:
 
 **Note**: It is possible to migrate the OS removing/adding groups of nodes, not only for a single node. However, be careful with the selected group of nodes - incorrectly selected nodes for removal or their amount can damage the cluster or lead it to an unusable state. Select the nodes at your discretion.
 
+**Warning** : Please use the `cache_version=False` option in `cluster.yaml` during the migration procedure
```

Comment:
Could you describe step-by-step what is happening with packages during OS migration and why exactly cache_version=False is necessary? With links to source code and explanation what fails.

Reply:
To be implemented in #252